.vscode/launch.json

```diff
@@ -25,6 +25,12 @@
       "presentation": {
         "hidden": true
       }
+    },
+    {
+      "type": "lldb",
+      "request": "attach",
+      "name": "Debug Electron Renderer",
+      "pid": "${command:pickProcess}"
```

Comment:
Interesting. Does this actually allow debugging the native code?

Reply:
Yep!

Reply:
Ooooooh